beacon-chain/core/blocks/payload.go

```diff
@@ -233,8 +233,14 @@ func verifyBlobCommitmentCount(body interfaces.ReadOnlyBeaconBlockBody) error {
 	if err != nil {
 		return err
 	}
-	if len(kzgs) > field_params.MaxBlobsPerBlock {
-		return fmt.Errorf("too many kzg commitments in block: %d", len(kzgs))
+	if body.Version() >= version.Electra {
```

Comment:
This change is related to: https://github.com/ethereum/consensus-specs/pull/4023
It's a place holder to make sure we can pass spec test, it'll be superseded by #14678